src/Tribe/Admin/Views.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+
+/**
+ * Class Tribe__Tickets__Admin__Views
```

Comment:
that is what I meant, yep - in that case, let's leave it (in my own experience I found anything ranging from a full on IDE to something like Sublime could figure out the class names from the definition itself, but if it's helpful to someone out there it's all good).